biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/GenbankReaderTest.java

```diff
@@ -163,6 +164,27 @@ public void testProcess() throws Exception {
 		assertEquals(3, dnaSequence.getAccession().getVersion().intValue());
 		assertTrue(genbankDNA.isClosed());
 	}
+	
+	@Test
+	public void testSequenceStream() {
+		CheckableInputStream inStream = new CheckableInputStream(this.getClass().getResourceAsStream("/two-dnaseqs.gb"));
```

Comment:
Did you also include this file in the test resources directory? It seems not to be in this PR